requirements/requirements.in

```diff
@@ -4,7 +4,7 @@ jsonobject==0.9.9
 jsonobject-couchdbkit~=0.9
 # celery 4.1.1 forked to work around https://github.com/celery/celery/issues/4737
 # upgrade when https://github.com/celery/celery/issues/4980 is fixed
--e git+git://github.com/nickpell/celery@dc7253614fd8500af78b09d1ceb79c50f1baac8d#egg=celery
+git+git://github.com/nickpell/celery@dc7253614fd8500af78b09d1ceb79c50f1baac8d#egg=celery
```

Comment:
should this have a version that goes with it? `egg=celery==4.1.1b1` or something like that?

Reply:
Should it?  I know we hypothesized this earlier but it seems to be working fine without it.

Reply:
so long as the previous releases continue to work it seems fine. TBH I'm not super sure why I included that with architect